conda_env/installers/pip.py

```diff
@@ -1,10 +1,19 @@
 from __future__ import absolute_import
+
 import subprocess
+from itertools import chain
+from os.path import join
 
+import conda.config as config
 from conda.cli import main_list
 
 
 def install(prefix, specs, args, env):
-    pip_cmd = main_list.pip_args(prefix) + ['install', ] + specs
+    # This mess is necessary to get --editable package sent to subprocess
+    # properly.
+    specs = list(chain(*[s.split() if '-e' in s else [s] for s in specs]))
+    # Directory where pip will store VCS checkouts for editable packages
+    src_dir = join(config.envs_dirs[0], env.name, 'src')
```

Comment:
`config.envs_dirs[0]` should be `$CONDA_ROOT/envs`, and I'm joining that with the name of the environment and `src`, so the final path will be `$CONDA_ROOT/envs/$CONDA_ENV/src`.  That's exactly what we want.